projects/adrv9009/src/app/headless.c

```diff
@@ -480,6 +480,8 @@ int main(void)
 
 #ifdef IIO_SUPPORT
 #ifdef ADRV9008_2
+	no_os_mdelay(100);
```

Comment:
maybe this delay deserves and inline comment so that we know why de delay is required.